### ISSUE TITLE
setAlpha method on Paint

### ISSUE THREAD

**Issue by renancaraujo** [flutter/flutter#25969]
The Skia Paint class has a `setAlpha` method that could be useful when drawing images into a canvas with transparency. We do not have a corresponding method in the Paint in Flutter. 

Is there any reason for that?

I know there is a workaround to use opacity in Paint:
```dart
Paint()..color = Color(0xFFFFFFFF).withAlpha(alpha);
```

**Comment by dnfield** [comment]
Alpha is controlled by the `Color` property for Flutter.  You can do it as you suggested, or simply include the alpha bits initially in your Color declaration, i.e. 

```
Paint()..color = Color(0x10FFFFFF);
```

The preference for engine API is to keep the surface as small and simple as possible.  See also @jason-simmons comment on your PR.

I'm going to close this for now, but please feel free to re-open if there's something you really think we're missing here!

**Comment by dnfield** [comment]
See also the `.fromARGB` and `.fromRGBO` constructors for `Color`: https://docs.flutter.io/flutter/dart-ui/Color/Color.fromARGB.html

e.g. `= Color.fromARGB(alpha, 255, 255, 255)`

**Comment by renancaraujo** [comment]
@dnfield  I see you aim to have the engine API simple. I figured the need for something like this when I had to draw an Image with some opacity. 
I know that the alpha channel is controlled by `color`, but having to set it just to achieve opacity looked a little bit weird, even more when drawing an image. 
After reading Skia docs I thought bringing [the setAlpha method](https://skia.org/user/api/SkPaint_Reference#SkPaint_setAlpha) to Flutter would codes like mine more undestandable.
But if it would bring more unecessary complexity on Flutter, I undestand that it for the better that  the API stay as it is right now.
